case-studies/eshoppen.md

```diff
@@ -153,6 +153,50 @@ For this component:
 * RR: A virtual machine with 2 vCPUs was used, this data was sourced from [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
 * TR: The bare metal host is split up into 16 vCPUs in total. This data was sourced from the [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
 
+
+### Front End Browser for Web application
+
+### Energy (`E`)
+
+The Quantification method used for calculating energy value is **Calculate**. We are measuring CPU utilization of the laptop client device and then using a model based on the Thermal Design Power (TDP) of the processors, number of cores etc to **estimate** the power consumption.
```

Comment:
A normal browser interaction by a single user will (a) last a length of time (b) be mostly zero CPU utilisation, this isn't a compute heavy task on the browser side.

Reply:
I think there are models already in use that map "length of time for browser session" to energy since it's mostly.